src/utils/components/Labels/WindowsLabel.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Label } from '@patternfly/react-core';
+import { WindowsIcon } from '@patternfly/react-icons';
+
+import './labels.scss';
+
+const WindowsLabel: FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Label variant="outline" className="os-label" icon={<WindowsIcon />}>
+      {t('Windwos only')}
```

Comment:
should be `Windows` :)

Reply:
thank you :)

Reply:
@avivtur are you sure it should be only "Windows"? IMHO "Windows only" is better than just "Windows". If I was a default normal user, seeing "Windows" would tell me nothing about that it is related to ONLY Windows.

Reply:
I meant she misspelled the word Windows :)